PetitParser2-PEG/PP2PEGGrammarTest.class.st

```diff
@@ -0,0 +1,218 @@
+"
+I am a set of generic tests for the PEG production rules.
+"
+Class {
+	#name : #PP2PEGGrammarTest,
+	#superclass : #TestCase,
```

Comment:
Can't you make this subclass of `PP2AbstractParserTest`?

Reply:
The PP2AbstractParserTest provides methods such as:
#assert:parse:
#assert:fail:

Moreover `PP2CompositeNodeTest` provides support for rules such as:
#parse:rule
#fail:rule

See `PP2ExpressionGrammarTest` and `PP2ExpressionParserTest` for examples.

Or did I miss something that prevents you from doing so?